File: Assignment 1/Java version/Convert.java
import java.util.Scanner;

public class Convert {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("Temperature in Fahrenheit: ");
		double input = scan.nextDouble();
		scan.close();
		double celcius = Math.round(((input - 32) * 5) / 9);
		System.out.println("Temperature in Celsius: " + celcius);
	}
}

File: Assignment 1/C# version/Convert.cs
using System;

class Convert
{
	public static void Main(string[] args)
	{
		Console.Write("Temperature in Fahrenheit: ");
		double input = System.Convert.ToDouble(Console.ReadLine());
		double celcius = Math.Round(((input - 32) * 5) / 9);
		Console.WriteLine("Temperature in Celsius: " + string.Format("{0:0.0}", celcius));
	}
}

